corehq/apps/hqadmin/static/hqadmin/js/visualizations.js

```diff
@@ -99,7 +99,8 @@ var HQVisualizations = function (options) {
             $(self.chart_tabs_id).on('click', 'a[data-toggle="hash-tab"]', function(){
                 $('.nvd3-chart').hide();
                 var $chart = $($(this).attr('href')).children('.nvd3-chart');
-                $chart.show();
+                $chart.show().removeClass('hidden');
```

Comment:
would be a good future refactor to not have to both use show/hide and toggle `.hidden`